project/Win32BuildSetup/genNsisInstaller.nsi

```diff
@@ -331,9 +331,44 @@ SectionEnd
 SectionGroupEnd
 
 Function .onInit
+  ; WinVista SP2 is minimum requirement
   ${IfNot} ${AtLeastWinVista}
-    MessageBox MB_OK|MB_ICONSTOP|MB_TOPMOST|MB_SETFOREGROUND "Windows Vista or above required.$\nThis program can not be run on Windows XP"
```

Comment:
`${IfNot} ${AtLeastWinVista}` condition will be true on XP and then the message below will be displayed.